src/engine/server/sv_client.cpp

```diff
@@ -41,7 +41,7 @@ Maryland 20850 USA.
 #include <common/FileSystem.h>
 
 Cvar::Cvar<bool> sv_wwwDownload("sv_wwwDownload", "server does a www dl redirect", Cvar::NONE, "");
-Cvar::Cvar<std::string> sv_wwwBaseURL("sv_wwwBaseURL", "base URL for redirect", Cvar::NONE, "");
+Cvar::Cvar<std::string> sv_wwwBaseURL("sv_wwwBaseURL", "base URL for redirect", Cvar::NONE, "dl.unvanquished.net/pkg");
```

Comment:
So technically, it would be better to keep it with the define in Defs.h. SInce while Unvanquished is the only game that uses daemon, the idea is to appeal to other people to use daemon as well. It's less forking if people just provide their own Defs.h instead of having to touch random stuff all over the engine.

Reply:
good point.